class.jetpack.php

```diff
@@ -4776,11 +4776,25 @@ public static function delete_secrets( $action, $user_id ) {
 	 * @since 2.6
 	 * @return int
 	 **/
-	public function get_remote_query_timeout_limit() {
```

Comment:
Since we're effectively deleting and creating a new method here, we should probably update the `@since`.